opentelemetry-api/src/opentelemetry/_logs/_internal/__init__.py

```diff
@@ -273,7 +273,7 @@ def get_logger(
     instrumenting_library_version: str = "",
     logger_provider: Optional[LoggerProvider] = None,
     schema_url: Optional[str] = None,
-    attributes: Optional[Attributes] = None,
+    attributes: LogAttributes = None,
```

Comment:
any reason not to have them as `Optional` like other params?

Reply:
They are, it's just that it's not repeated :) `LogAttributes = Optional[Mapping[str, "AnyValue"]]`

Reply:
Yeah, but that's not really common. This usually leads to a wrong type inside the function, because you usually want different logic between `None` (default), and the actual type.